src/isomorphic/ReactLifeCycleTimers.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactLifeCycleTimers
+ */
+
+'use strict';
+
+var keyMirror = require('keyMirror');
+
+var ReactLifeCycleTimers = keyMirror({
+  COMPONENT_DID_MOUNT: null,
+  COMPONENT_DID_UPDATE: null,
+  COMPONENT_WILL_MOUNT: null,
+  COMPONENT_WILL_RECEIVE_PROPS: null,
+  COMPONENT_WILL_UNMOUNT: null,
+  COMPONENT_WILL_UPDATE: null,
+  CONSTRUCTOR: null,
+  UPDATE_REFS: null,
+  RENDER: null,
+  SHOULD_COMPONENT_UPDATE: null,
+});
```

Comment:
I changed lifecycle timers to use the method name (`render`, `componentDidMount`, etc, except for `updateRefs` which isn’t a real name—couldn’t figure out a better way to phrase this), and the reconciler timers to use the internal method name (e.g. `mountComponent`).